message Server
{
	optional string name = 1;
	optional int32 status = 2;
	optional int32 type = 3;
}

message ProtocolWrapper
{
	optional int32 protocol_id = 1;
	optional int32 client_id = 2;
	optional bytes inner_protocol = 3;
}